.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
I _think_ this should be removed. It's certainly not necessary, and I don't see any of the examples in the docs starting with this.